src/Sushi.MicroORM.FakeAssemblyTests/TestFakes.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Sushi.MicroORM.Mapping;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sushi.MicroORM.FakeAssemblyTests
+{
+    public class TestClass
```

Comment:
Please also add a test fake datamap which is not a subclass, and prove the scanner can also find it